docs/workflow_tasks/queue_granules.md

```diff
@@ -0,0 +1,94 @@
+---
+id: queue_granules
+title: Queue Granules
+hide_title: false
+---
+
+This task utilizes the Cumulus Message Adapter to interpret and construct incoming and outgoing messages.
+
+Links to the npm package, task input, output and configuration schema definitions, and more can be found on the auto-generated [Cumulus Tasks](../tasks) page.
+
+## Summary
+
+The purpose of this task is to schedule ingest of granules that were discovered on a remote host, whether via the [DiscoverGranules](./discover_granules) task or the [ParsePDR](./parse_pdr) task.
+
+The task utilizes a defined [collection](../data-cookbooks/setup#collections) in concert with a defined [provider](../data-cookbooks/setup#providers), either on each granule, or passed in via config to queue up ingest executions for each granule, or for batches of granules.
```

Comment:
The links for `collection` and `provider` aren't valid if I run the docs locally. I think they should be going to this page:

https://nasa.github.io/cumulus/docs/configuration/data-management-types

Reply:
Refactoring.